lib/jxl/enc_frame.cc

```diff
@@ -1647,7 +1647,8 @@ bool CanDoStreamingEncoding(const CompressParams& cparams,
   if (cparams.max_error_mode) {
     return false;
   }
-  if (cparams.modular_mode) {
+  if (cparams.modular_group_size_shift != 1 &&
+      cparams.modular_group_size_shift != -1) {
```

Comment:
now if (metadata.m.num_extra_channels > 0) condition should be removed below, otherwise we would attempt to streaming encode modular, responsive images without alpha.

Reply:
Not quite - I believe it should be changed to `metadata.m.num_extra_channels > 0 || cparams.modular_mode`.